charts/spire/charts/spire-server/values.yaml

```
@@ -372,11 +368,35 @@ tornjak:
     successThreshold: 1
     timeoutSeconds: 5
 
+  # tornjak - Tornjak specific configuration
   config:
-    # -- persistent DB for storing Tornjak specific information
+    # -- Persistent DB for storing Tornjak specific information
     dataStore:
       driver: "sqlite3"
       file: "/run/spire/data/tornjak.sqlite3"
+
+    # Tornjak supports 3 connection types: `http`, `tls`, and `mtls`.
+    # When `tlsSecret` is created in this chart namespace, the TLS connection is started
+    # When `tlsSecret` and `userCa.tornjak-user-ca` are created in this chart namespace, the mTLS connection is started
+    # When none of them are created, Tornjak starts with HTTP connection only
+
+    # -- Name of the secret containing server side key and certificate for TLS verification
+    # (required for `tls` or `mtls` connectionType)
+    tlsSecret: tornjak-tls-secret
+    userCA:
+      # -- Type of delivery for the user CA for mTLS client verification
+      # options are `Secret` or `ConfigMap`
+      # (required for `mtls` connectionType)
+      type: Secret
+      name: tornjak-user-ca
+
+    # -- Enables the service for a given `connectionType`
+    service:
+      type: ClusterIP
+      portHttp: 10080
+      portHttps: 10443
+      annotations: {}
```

The comment is a bit misleading as this is the k8s service. For other native k8s service object values we don't have documentation explaining the service either.

Reducing the indentation level as suggested above, to keep it similar like the remainder of charts.

```suggestion
  service:
    type: ClusterIP
    portHttp: 10080
    portHttps: 10443
    annotations: {}
```